src/vaadin-menu-bar-buttons-mixin.html

```diff
@@ -99,6 +99,7 @@
     }
 
     _onItemClick(e) {
+      e.preventDefault();
```

Comment:
Why is this needed?

Reply:
That is related to the issue described in https://github.com/vaadin/vaadin-menu-bar/issues/9#issuecomment-477059015

> ↓ (arrow down) should not scroll window when opening sub-menu

By default, arrow down scrolls the page, which shouldn't happen with menu-bar.

Reply:
I see. To improve code readability the function name could be changed (I had the impression that this was called only on click events based on the name). I don't have good suggestions though... :D Not blocking.